spring-batch-infrastructure/src/test/java/org/springframework/batch/item/file/mapping/BeanWrapperFieldSetMapperTests.java

```diff
@@ -457,6 +460,9 @@ public void testConversionWithTestConverter() throws Exception {
 	@Test
 	public void testDefaultConversion() throws Exception {
 
+		TimeZone defaultTimeZone = TimeZone.getDefault();
+		TimeZone.setDefault(UTC_TIME_ZONE);
```

Comment:
This should be done in a `@Before` and reset in an `@After`

Reply:
Fixed as requested. Thank you for the review.